src/jit/compiler.cpp

```diff
@@ -829,14 +800,21 @@ var_types Compiler::getArgTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
         //
         if (structSize <= MAX_PASS_MULTIREG_BYTES)
         {
-            // Structs that are HFA's are passed by value in multiple registers
-            if (IsHfa(clsHnd)
+            // Structs that are HFA's are passed by value in multiple registers.
```

Comment:
Shouldn't this be `structs that are HFAs/HVAs`?

Reply:
I'll update the comment.